src/precompiles/ArbGasInfo.sol

```diff
@@ -129,4 +129,24 @@ interface ArbGasInfo {
 
     /// @notice Returns the available funds from L1 fees
     function getL1FeesAvailable() external view returns (uint256);
+
+    /// @notice Returns the equilibration units parameter for L1 price adjustment algorithm
+    /// Available in ArbOS version 20
+    function getL1PricingEquilibrationUnits() external view returns (uint256);
+
+    /// @notice Returns the last time the L1 calldata pricer was updated.
+    /// Available in ArbOS version 20
+    function getLastL1PricingUpdateTime() external view returns (uint64);
+
+    /// @notice Returns the amount of L1 calldata payments due for rewards (per the L1 reward rate)
+    /// Available in ArbOS version 20
+    function getL1PricingFundsDueForRewards() external view returns (uint256);
+
+    /// @notice Returns the amount of L1 calldata posted since the last update.
+    /// Available in ArbOS version 20
+    function getL1PricingUnitsSinceUpdate() external view returns (uint64);
+
+    /// @notice Returns the L1 pricing surplus as of the last update (may be negative).
+    /// Available in ArbOS version 20
+    function getLastL1PricingSurplus() external view returns (int256);
```

Comment:
what is the difference between this and `getL1PricingSurplus()`?

Reply:
This returns the value as of the last update, which is internal state that might be useful for debugging the pricer